### ISSUE TITLE
feature request: upload images

### ISSUE THREAD

**Issue by alexgleason** [Matterwiki/Matterwiki#18]

**Comment by nshntarora** [comment]
Yes, this is one of the most requested features as of now. Can't guarantee it in 0.2, but for sure by 0.3

**Comment by frdmn** [comment]
Any updates on this? Would love to have a image upload feature as well :)

**Comment by nshntarora** [comment]
Yes, we're already working on a new text editor. Here is the pull request to track the progress.

#74 

@krishnagopinath is leading the entire editor overhaul. Let him know if you have any requests/suggestions.

:smile:

**Comment by Rohanhacker** [comment]
Any updates on this ?

**Comment by akella** [comment]
No news? :(

**Comment by mlgarchery** [comment]
Keep up the good work ! good luck

**Comment by ghost** [comment]
Yes
